Felienne/hedy
# [FEATURE] Implement multi-language mail templates

**Description**
In this PR we implement multi-language mail templates. Currently we only support English mails, but especially for children it would be nice to also support their selected language. We also add the username to the mail body to make sure users don't forget it.

**Fixes**
This PR fixes #2125.

**How to test**
We are unable to test this locally. Deploy to Alpha and verify that the mail templates work in both English and dutch.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
